api/v1alpha1/groupversion_info.go

```diff
@@ -0,0 +1,36 @@
+/*
+
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package v1alpha1 contains API Schema definitions for the nmstate.io v1alpha1 API group
```

Comment:
In all the new CRs, you moved the documentation below all the tags. Why not here?

Reply:
This is how it looks if you create an empty project from scratch with operator-sdk and then you add an api, looks like it's placing tags differently.

Reply:
Ack